### ISSUE TITLE
Search depth in the library

### ISSUE THREAD

**Issue by joelsleeba** [jghauser/papis.nvim#37]
Hi. Thank you for the amazing plugin.
Currently my papis library is structed like below
```
papis_library
├── author 1
│   └── paper 1
│       ├── info.yaml
│       └── author 1 - paper 1.pdf
├── author 2
│   ├── paper 1
│   │   ├── author 2 - paper 1.pdf
│   │   └── info.yaml
│   ├── paper 2
│   │   ├── author 2 - paper 2.pdf
│   │   └── info.yaml
│   └── paper 3
│       ├── author 2 - paper 3.pdf
│       └── info.yaml

```

When I set `dir = papis_library` this does not detect my `info.yaml` files. I got `papis.nvim` working if i set the `dir` to `papis_library/author 2`. But this gives me access to the articles by author 2. I guess this is because of the search depth implemented in the plugin.

Restructuring my library would resolve it. But I feel implementing a deeper search would be nice for folder structure arrangement.

**Comment by jghauser** [comment]
Hi, thanks for the issue report! You're right I probably didn't think of such library structures when I designed this plugin. This is definitely something that `papis.nvim` should support. I'll try have a look at it towards the end of next week -- there's a bunch of other things I've been wanting to get to as well!

**Comment by jghauser** [comment]
This should be fixed with #58 and `papis.nvim` should now recursively search paths of any depth. Let me know if anything still doesn't work. :)
